qa/tasks/vstart_runner.py

```diff
@@ -242,8 +242,11 @@ def _perform_checks_and_return_list_of_args(self, args, omit_sudo):
 
         # We'll let sudo be a part of command even omit flag says otherwise in
         # cases of commands which can normally be ran only by root.
+        cmds_that_need_root_access = ['-u', 'useradd', 'passwd', 'usermod',
+                                      'userdel', 'groupadd', 'groupmod',
+                                      'groupdel', 'chown']
         try:
-            if args[args.index('sudo') + 1] in ['-u', 'passwd', 'chown']:
+            if args[args.index('sudo') + 1] in cmds_that_need_root_access:
                 omit_sudo = False
```

Comment:
This isn't going to work. Think carefully here: we're going to run a test that is configuring users or groups. We definitely don't want to do that on our dev machine.

Which test is failing in vstart_runner.py that prompted this change?

Reply:
> This isn't going to work. Think carefully here: we're going to run a test that is configuring users or groups. We definitely don't want to do that on our dev machine.

Right. I was only thinking from the perspective of running the tests locally.

> Which test is failing in vstart_runner.py that prompted this change?

I was writing this patch because there can be a case where we might need multiple users to test. This happened to me when I was writing test for ACLs. But if you think this is very rare, this PR can be closed.

Reply:
In those cases, I think we'll just test in teuthology. vstart_runner.py will not work for every test.